jplazcano87/spaceghost-showcase-ios
# [Feature] Login UI

# Showcase app Pull Request
## Status

**READY**
## Description

Login UI for the Showcase app.
## Steps to Test or Reproduce

Run the app and you will see the new login 
![simulator screen shot 26-02-2016 12 09 12 a m](https://cloud.githubusercontent.com/assets/3168897/13342381/b336c586-dc1f-11e5-8a87-26ef9ae1484b.png)
